Added Tesla (v4.6.1) (#102)
riverar/wsa-app-compatibility: README.md

| TikTok (TV Version) | 1.6.0 | ❌ | App crashes ||
| TikTok Lite | 21.7.1 | ❌ | App crashes ||
| Telegram | 8.1.2 | ✅ |||

> | Tesla | 4.6.1 | 🆖 | Vehicle graphics do not load. Map does not load, cannot enable phone key. | Internet-based vehicle controls, charge stats, service all work. |

Reviewer: ```suggestion
| Tesla | 4.6.1 | 🆖 | Vehicle graphics and maps do not load, cannot enable phone key. | Internet-based vehicle controls, charge stats, services are functional. |
```
